### PR TITLE
Nc news 10

### PR DESCRIPTION
**Pull request by kctan0814**
Added functionality and test suite to GET request to this endpoint

### CODE REVIEW

File: app/controller/articles.controller.js [hunk 1]
```diff
@@ -17,4 +18,10 @@ exports.getCommentsByArticleId = (req, res, next) => {
   selectCommentsByArticleId(article_id).then((comments) => {
     res.status(200).send({comments})
   }).catch(next)
+}
+
+exports.getUsers = (req, res, next) => {
+  selectUsers().then((users) => {
+    res.status(200).send({users})
```
Comment (lewispricey) [inline app/controller/articles.controller.js]: Great choice of key to send the data back on 💯

File: app/__tests__/app.test.js [hunk 2]
```diff
@@ -146,4 +146,21 @@ describe('GET /api/articles', () => {
...
       })
   })
+})
+
+describe('GET /api/users', () => {
+  test('200: returns a list of all the users in the database', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({body: {users}}) => {
```
Comment (lewispricey) [inline app/__tests__/app.test.js:157]: This is a great test, before looping I'd deffinitely add in a check to ensure the response is an array and has a length more than 0 just so you can be confident this test won't false pass if the array is empty.

**Review by lewispricey — APPROVED** [review]
Great work here KC your code is super readable!

I've added a note on the code for a small addition to your test here to prevent any false passes but once this has been added please feel free to merge away 🚀
